src/main/c/Makefile

```diff
@@ -11,7 +11,7 @@ endif
 no_cexts: clean
 	@echo "WARNING: TRUFFLERUBY_CEXT_ENABLED is set to false - not building cexts" 1>&2
 
-all: $(ROOT)/lib/cext/ruby.su $(ROOT)/lib/mri/openssl.su $(ROOT)/lib/mri/zlib.su $(ROOT)/lib/mri/syslog.su
+all: $(ROOT)/lib/cext/ruby.su $(ROOT)/lib/mri/openssl.su $(ROOT)/lib/mri/zlib.su $(ROOT)/lib/mri/syslog.su $(ROOT)/lib/cext/trufflemock.bc
 
 clean:
 	rm -f $(ROOT)/lib/cext/ruby.su
```

Comment:
Ah, its done below, but it's better to keep them together since they are also generated by the same Makefile rule.